docker/common-services.yaml

```diff
@@ -73,6 +73,8 @@ services:
       FIFTYONE_APP_ANONYMOUS_ANALYTICS_ENABLED: ${FIFTYONE_APP_ANONYMOUS_ANALYTICS_ENABLED:-true}
       FIFTYONE_APP_DEFAULT_QUERY_PERFORMANCE: ${FIFTYONE_APP_DEFAULT_QUERY_PERFORMANCE}
       FIFTYONE_APP_ENABLE_QUERY_PERFORMANCE: ${FIFTYONE_APP_ENABLE_QUERY_PERFORMANCE}
+      FIFTYONE_APP_BANNER_TEXT: ${FIFTYONE_APP_BANNER_TEXT}
+      FIFTYONE_APP_BANNER_COLOR: ${FIFTYONE_APP_BANNER_COLOR}
```

Comment:
don't put these here unless they're required

People can optionally add arbitrary env vars to their compose.override.yaml file